deploy/kubernetes/operator.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: deployment-validation-operator
+  name: deployment-validation-operator
+  namespace: deployment-validation-operator
+spec:
+  replicas: 3
```

Comment:
We know that more than 1 replica doesn't currently work, may be worthwhile to reduce to 1 until that's fixed.

Reply:
Good catch.